packages/terra-clinical-result/src/_BloodPressureDisplay.jsx

```diff
@@ -46,7 +56,20 @@ const BloodPressureDisplay = ({
     <ConditionalWrapper
       key={`del-${type}-${result.eventId}`}
       condition={result.statusInError}
-      wrapper={children => <del>{children}</del>}
+      wrapper={children => (
+        <span>
+          <s aria-hidden="true">{children}</s>
+          {isValidValue
+            ? (
+              <VisuallyHiddenText text={intl.formatMessage(
+                { id: 'Terra.clinicalResult.statusInErrorStrikethrough' },
+                { strikethroughResult: `${result.result.value} ${!hideUnit && result.result.unit ? result.result.unit : ''}` },
```

Comment:
Nit: Since the hideUnit check was added, would you mind pulling the whole expression out and making it a variable?

Reply:
Good idea, changed here: [6c8a6ad](https://github.com/cerner/terra-clinical/pull/870/commits/6c8a6add181de629fe965e6dbc78fee621666359)